website_blog_tags_variable_dimension/models/blog_tag.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+import random
+
+class BlogTag(models.Model):
+    _inherit = 'blog.tag'
+
+    # doing it for all blogs. need to make it per-blog.
+    min_font = 10.0
+    max_font = 19.0
+
+    @api.multi
+    def get_blog_most_used_tag_cardinality(self):
+        tags = self.env['blog.tag'].search([])
```

Comment:
@susport, attention, this is a particular function.  "self" could be  any recordset , I need to actually scan all tags. this function has to return max cardinality amongst all records.. so unfortunately the more elegant  expression is not possible, the search needs to happen.